datadog_checks_dev/datadog_checks/dev/tooling/commands/manifest.py

```diff
@@ -67,21 +60,27 @@ def manifest():
 def verify(fix, include_extras):
     """Validate all `manifest.json` files."""
     all_guids = {}
-    failed = 0
+    task_failed = False
 
     root = get_root()
     root_name = basepath(get_root())
 
+    ok_checks = 0
+    failed_checks = 0
+    fixed_checks = 0
+    echo_info("Validating all manifest.json files...")
     for check_name in sorted(os.listdir(root)):
         manifest_file = os.path.join(root, check_name, 'manifest.json')
 
         if file_exists(manifest_file):
-            display_queue = [(echo_info, '{} ->'.format(check_name))]
+            display_queue = []
+            file_failures = 0
+            file_fixed = False
```

Comment:
Per above comment, we don't need any of this logic

Reply:
I do need this one to know if a file had failures and has been fixed or if it was ok all along.